<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generador</class>
 <widget class="QWidget" name="Generador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>529</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1112</width>
    <height>529</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1112</width>
    <height>529</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1112</width>
    <height>529</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 78, 115);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(227, 227, 227);</string>
   </property>
   <property name="text">
    <string>Generador de Llaves</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(106, 213, 158);</string>
   </property>
   <property name="text">
    <string>Primer Número Primo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(106, 213, 158);</string>
   </property>
   <property name="text">
    <string>Segundo Número Primo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>170</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(106, 213, 158);</string>
   </property>
   <property name="text">
    <string>Primer Número Primo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="boxPrimer">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(209, 209, 209);
border-color: rgb(88, 88, 132);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="boxSegun">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(209, 209, 209);
border-color: rgb(88, 88, 132);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="boxExpo">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(209, 209, 209);
border-color: rgb(88, 88, 132);
</string>
   </property>
   <property name="text">
    <string>65537</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerarPublica">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 194, 0);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Generar Llave Pública</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerarPrivada">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 91, 91);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Generar Llave Privada</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerarPrimos">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 145, 0);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Generar Números</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorPrimo1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorPrimo2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorPrimo3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegresar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 182, 223);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
